systemtest/src/test/resources/connect-build/build-connect-image.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+RANDOM=$(shuf -i 60000-99999 -n 1)
+
+KAFKA_VERSION=$1
+BASE_IMAGE=$2
+CONNECT_IMAGE=${3:-"strimzi/connect:$RANDOM"}
```

Comment:
I don't get the random part. If it has some reason, it should have some comment explaining it. But it seems like a stable fixed name would be better. You push this only into the test registry that should be limited to the CI, so no need for randomness.

Reply:
Yeah, you are right.
At the beginning I thought that it will be better to have the tag randomized, so we will ensure that we are always using the "latest" or "correct" one, but this is the case for Connect build tests and not this one. I will remove that, thanks.